.gitignore

```diff
@@ -83,6 +83,8 @@ Makefile
 cmake_install.cmake
 CMakeFiles
 CMakeCache.txt
+# cmake files generated by CLion
+/cmake-build-debug
```

Comment:
It's better to use `/cmake-build-*` just like what JetBrains suggested: 
https://github.com/github/gitignore/blob/4488915eec0b3a45b5c63ead28f286819c0917de/Global/JetBrains.gitignore#L43-L44